### PR TITLE
Completed Assessmentfor week 1

### PR DESCRIPTION
**Pull request by oagomezz**
# Submitting Week 1 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [ x] Coding Challenges
- [ x] Coding Concepts
- [x ] Interview Questions

### 📝 Feedback

If you would like specific feedback on a particular problem please leave details here:

### CODE REVIEW

File: interview-questions.md [hunk 1]
```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
...
+Researched answer:Git is a tool used as a revision control system that can be used to track changes or updates to a program. It helps mitigate obsolete revisions and assists program developers to backtrack if the code starts to get buggy. Github is a cloud based repository that helps developers store and manage their progams along with any changes that were made. 
 
-2. Which JavaScript operators will return a Boolean value?
+2. Which JavaScript operators will return a Boolean value? 
 
-Your answer:
+Your answer: logical operators since the conditions are based off booleans, with each respective answer will return a different output. 
 
-Researched answer:
+<!-- Researched answer:   -->
```
Comment (TsunamiCode5) [inline interview-questions.md]: This research answer is missing, please take a look at this

**Review by TsunamiCode5 — CHANGES_REQUESTED** [review]
I left a comment on the second question in interview-questions

**Review by oagomezz — COMMENTED** [review]
Made the corrections requested.

File: interview-questions.md [hunk 2]
```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
...
+Researched answer:Git is a tool used as a revision control system that can be used to track changes or updates to a program. It helps mitigate obsolete revisions and assists program developers to backtrack if the code starts to get buggy. Github is a cloud based repository that helps developers store and manage their progams along with any changes that were made. 
 
-2. Which JavaScript operators will return a Boolean value?
+2. Which JavaScript operators will return a Boolean value? 
 
-Your answer:
+Your answer: logical operators since the conditions are based off booleans, with each respective answer will return a different output. 
 
-Researched answer:
+Researched answer: the logical operators that will return a boolean value will be the && operator or logical and which requires both operands to be true to evaluate the condition as true. The || operator requires one of the operands to be true in order to evaluate the condition as true the final logical operator is the bang (!) which returns the opposite of the value for example value != 0 returns for any value not equal to 0.
```
Comment (TsunamiCode5) [inline interview-questions.md:25]: Beautiful!

**Review by TsunamiCode5 — APPROVED** [review]
Oscar you did awesome on this assessment. You had great pseudo code and good explanations in the coding-concepts. Your functions look clean and you put a lot of effort into answering your interview questions. They are good practice and real potential questions an interviewer may ask you! This assessment comprehension level is advanced beginner, this is great work!

**Pull request merged**